client/src/pages/SubmitDesign.js

```diff
@@ -107,6 +109,8 @@ export default function SubmitDesign(props) {
             .then(response => response.json())
             .then(data => {
                 console.log('Success:', data);
+                alert("Successfully upload the image")
+                history.goBack()
```

Comment:
Instead of going back, lead them to the link because if you visit the submission page directly, you won't have any history to go back. Hold on making this change because we need to make the /submit-design route to be specific to a contest rather than passing in an ID into this page. If you refresh /submit-design, you won't be able to identify the contest